eng/pipelines/templates/jobs/ci.yml

```diff
@@ -218,6 +219,43 @@ jobs:
         displayName: Build snippets
         condition: and(succeeded(), eq(${{ parameters.BuildSnippets }}, true))
       - template: /eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml
+  - ${{ if ne(parameters.TestDependsOnDependency, 'not-specified') }}:
+    - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
+      parameters:
+        GenerateJobName: generate_code_generation_matrix 
+        JobTemplatePath: /eng/pipelines/templates/jobs/code.gen.yml
+        MatrixConfigs:
+        - Name: NET_ci_codegen
+          Path: eng/pipelines/templates/stages/code-gen-matrix.json
+          Selection: sparse
+          GenerateVMJobs: true
+        MatrixFilters: ${{ parameters.MatrixFilters }}
+        MatrixReplace: ${{ parameters.MatrixReplace }}
+        CloudConfig:
+          Cloud: public
```

Comment:
I know you are trying to reuse as much as possible for generating the matrix bot some parameters like this don't really apply but is it still needed?

Reply:
Removed the unneccessary parts.